examples/templates/docker/main.tf

```diff
@@ -11,62 +11,18 @@ terraform {
   }
 }
 
-# Admin parameters
-
-# Comment this out if you are specifying a different docker
-# host on the "docker" provider below.
-variable "step1_docker_host_warning" {
-  description = <<-EOF
-  This template will use the Docker socket present on
-  the Coder host, which is not necessarily your local machine.
-
-  You can specify a different host in the template file and
-  suppress this warning.
-  EOF
-  validation {
-    condition     = contains(["Continue using /var/run/docker.sock on the Coder host"], var.step1_docker_host_warning)
-    error_message = "Cancelling template create."
-  }
-
-  sensitive = true
-}
-variable "step2_arch" {
-  description = <<-EOF
-  arch: What architecture is your Docker host on?
-
-  note: codercom/enterprise-* images are only built for amd64
-  EOF
-
-  validation {
-    condition     = contains(["amd64", "arm64", "armv7"], var.step2_arch)
-    error_message = "Value must be amd64, arm64, or armv7."
-  }
-  sensitive = true
-}
-variable "step3_OS" {
-  description = <<-EOF
-  What operating system is your Coder host on?
-  EOF
-
-  validation {
-    condition     = contains(["MacOS", "Windows", "Linux"], var.step3_OS)
-    error_message = "Value must be MacOS, Windows, or Linux."
-  }
-  sensitive = true
+data "coder_provisioner" "me" {
 }
 
 provider "docker" {
-  host = var.step3_OS == "Windows" ? "npipe:////.//pipe//docker_engine" : "unix:///var/run/docker.sock"
-}
-
-provider "coder" {
+  # host = data.coder_provisioner.me.os == "windows" ? "npipe:////.//pipe//docker_engine" : "unix:///var/run/docker.sock"
```

Comment:
why is this commented out? this works on MacOS but not sure if it would on Windows

Reply:
Oops, wasn't intentional. Fixed!

Reply:
Cool - confirmed it still works